src/XrdPss/XrdPssTrace.hh

```diff
@@ -39,8 +39,12 @@
 
 #define QTRACE(act) SysTrace.What & TRACEPSS_ ## act
 
+#define TRACING(x) (SysTrace.What & x)
+
+#define DEBUGON TRACING(TRACEPSS_Debug)
+
 #define DEBUG(tid,y)\
-    if (SysTrace.What & TRACEPSS_Debug) SYSTRACE(SysTrace., tid, epname, 0, y)
+    if (DEBUGON) SYSTRACE(SysTrace., tid, epname, 0, y)
```

Comment:
Isn't `if(DEBUGON)` redundant if it's already done within the `DEBUG` macro?

Reply:
I see it's because of usage of `obfuscateAuth`, so the redundant use is ok. Thanks!